xapian-core/tests/api_closedb.cc

```diff
@@ -31,14 +31,18 @@
 using namespace std;
 
 #define COUNT_CLOSEDEXC(CODE) \
-    try { CODE; } catch (const Xapian::DatabaseError &) { ++closedexc_count; }
+    try { \
+		CODE; \
+	} catch (const Xapian::DatabaseClosedError &) { \
+		++closedexc_count; \
```

Comment:
This line and `CODE; \` above are overindented - each indent level is generally 4 columns in Xapian code, but here you've used 8.

Reply:
Thanks for pointing this out.I will push the changes.